golem-worker-executor-base/src/services/scheduler.rs

```diff
@@ -135,6 +161,10 @@ impl SchedulerServiceDefault {
         let previous_hour_key = Self::schedule_key_from_timestamp(previous_hours_since_epoch);
         let current_hour_key = Self::schedule_key_from_timestamp(hours_since_epoch);
 
+        // TODO: couple of issues with this implementation
```

Comment:
These were all already present in the current implementation, this is just notes for later when we do a cleanup pass. I think it's fine for now for our purposes